src/datasets/utils/resources/languages.json

```diff
@@ -6,8 +6,10 @@
     "agq-CM": "Aghem (Cameroon)",
     "ak": "Akan",
     "ak-GH": "Akan (Ghana)",
+    "als": "Tosk Albanian",
```

Comment:
Why are you adding languages in this PR? 😕

Reply:
I can make a separate PR if you want.
Basically these are the missing languages from OSCAR, which is a dataset that required minor changes to be compatible with streaming. Because of the change it triggered the metadata validation in the CI so I fixed the metadata, i.e. the languages.
The nez codes are ISO 639-3 codes and don't have ISO 639-1 nor ISO 639-2 codes.